lntest/itest/list_on_test.go

```diff
@@ -16,8 +16,32 @@ var allTestCasesTemp = []*lntemp.TestCase{
 		TestFunc: testBasicChannelFunding,
 	},
 	{
-		Name:     "test multi-hop htlc",
-		TestFunc: testMultiHopHtlcClaims,
+		Name:     "multi hop htlc local timeout",
```

Comment:
Ah nice, this will spread the duration of the individual tests much better across the parallel running tranches! With all your improvements we might even be able to increase the parallelism to 6 or 8 concurrent tests (need to experiment with that).
So smaller/shorter tests are way better :tada:

Reply:
I tried with 8 locally but it seemed to slow down the test quite a bit, maybe it's my machine not dealing with goroutine schedulers smart enough🧐. Anyway totally agree with smaller/shorter tests are better. I find them not only run faster but also easier to debug.